tests/test_authentication.py

```diff
@@ -46,6 +48,16 @@ def test_login_logout(client, user):
     assert ret.json['status'] == 'error'
     assert ret.json['message'] == 'invalid user or password'
 
+    ret = client.post('/api/logout/')
+    assert ret.status_code == 200
+
+    ret = client.post('/api/login/', data={**LOGIN_DATA, 'remember_me': True})
+    assert ret.status_code == 200
+    # with remember_me, we should have two cookies
+    assert len(client.cookie_jar) == 2
+
+    ret = client.post('/api/logout/')
```

Comment:
Nothing is tested on this `ret`, should be removed?

Reply:
Yep, just so the next tests have no logged in user.

Reply:
Added a comment